docs/conf/portland/2020/example-proposal.rst

```diff
@@ -0,0 +1,58 @@
+:template: {{year}}/generic.html
+
+Example talk proposal title
+===========================
+
+Substitute a catchy title that promises pratical lessons
+
+Abstract
+--------
+
+This is the content that Write the Docs would post on the conference schedule. The proposal committee uses everything beyond the abstract to evaluate the quality of your proposal.
+
+Include a brief story, typically two to four paragraphs, that describes your personal work experience with the topic. Make sure the abstract appeals to our audience of documentarians. We're suspicious of proposals that look like they belong at other conferences.
+
+Include a list of things that our audience can learn from your talk, such as:
+
+- Lessons that the audience can apply in their own work
+- Things that audiences should research further
+- Spoilers that provide details about the talk
+
+Help your audience think: "Oh yes, this talk could help me when I do X in my work!"
+
+Avoid walls of text. We recommend that you limit your proposal to a maximum of 300 words. If your proposal is accepted, this is what the audience will see in the program when they preview your talk.
+
+Who and Why
+-----------
+
+Use this opportunity to show the selection committee that you empathize with your audience. Help them think: "Oh yes, this talk could help me when I do X in my work!"
+
+Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that focus on a specific software tool.
```

Comment:
Suggestion:

> Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we are wary about accepting talks that focus on a specific software tool **or set of tools. Talks that discuss tools should also discuss the wider context, applications, and implications of implementing the tools.**

Reply:
I think this would invite more tooling talks, so I don't think this is wise. However,

@plaindocs If you think a substantial number of tooling talks are inevitable, @eirinikos 's option makes sense.

Reply:
Hmmm, I think a little leeway is a good thing. We could always roll back if it doesn't work out. :-D